utils/run-test

```diff
@@ -171,6 +171,7 @@ def main():
         # build_dir.
         paths = args.paths
         if not paths:
+            parser.print_usage()
             error_exit("error: too few arguments")
```

Comment:
Nit: is this error even needed anymore now that we show the usage? It seems redundant